### PR TITLE
[MIG][15.0] edi_storage_oca

### PR DESCRIPTION
**Pull request by i-vyshnevska**
depends on
- [X]  https://github.com/OCA/edi/pull/490
- [X]  https://github.com/OCA/connector/pull/425
- [x] https://github.com/OCA/storage/pull/145

### CODE REVIEW

**Review by gurneyalex — APPROVED** [review]

**Review by simahawk — CHANGES_REQUESTED** [review]
waitig for the base PR to get merged

**Comment by etobella** [comment]
@i-vyshnevska Can you rebase?

File: test-requirements.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
+odoo_test_helper @ git+https://github.com/akretion/odoo-test-helper@refs/pull/13/head#
```
Comment (simahawk) [inline test-requirements.txt]: this is wrong. It's released on pypi and it's in OCA BTW

Comment (simahawk) [reply test-requirements.txt]: https://pypi.org/project/odoo-test-helper/2.0.1/

File: test-requirements.txt [hunk 2]
```diff
@@ -1,2 +1,2 @@
-
-odoo_test_helper
+odoo-test-helper>=1.1.0
```
Comment (simahawk) [inline test-requirements.txt]: 2.0.1

Comment (simahawk) [reply test-requirements.txt]: storage_backed is getting merged

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM functional test

**Review by etobella — APPROVED** [review]

**Comment by i-vyshnevska** [comment]
@simahawk  ready to merge

**Review by simahawk — APPROVED** [review]

**Comment by etobella** [comment]
pre-commit failed on the Translation commit. Trying again !

/ocabot merge nobump

**Comment by etobella** [comment]
Ouch! Weird issue here 
```
Error: reserveCache failed: Cache already exists. Scope: refs/heads/15.0-ocabot-merge-pr-491-by-etobella-bump-nobump, Key: pre-commit-2-e233dd2121e64c861dfc8efd42bc49a71b455d4b158f53eb6973b5ed6703f473-566dc591541fabffba7ca60328d7d5fbd4f389739590050aa833e6a3e6d2b4ac, Version: 28cdb9f5496f334116f23e86f0063f5d3a9348c2e22425a33171e071aadada7e
```
@simahawk any idea?

**Pull request merged**